src/components/Switch/Switch.stories.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react'
+import { storiesOf } from '@storybook/react'
+import { MemoryRouter } from 'react-router'
+import Switch from 'components/Switch'
+import SwitchStyled from 'components/SwitchStyled'
+
+storiesOf('Switch', module)
+  .addDecorator(story => (
+    <MemoryRouter>{story()}</MemoryRouter>
+  ))
+  .add('Basic', () => <SwitchDemo />)
+  .add('SwitchStyled', () => <SwitchStyledDemo />)
+
+function SwitchDemo () {
+  const [switchBool, setSwitchBool] = useState(true)
+
+  function handleToggleSwitch () {
+    setSwitchBool(!switchBool)
+  }
+  return (<Switch value={switchBool} onClick={handleToggleSwitch} />)
+}
+
+function SwitchStyledDemo () {
+  const [switchBool, setSwitchBool] = useState(false)
+
+  function handleToggleSwitch (checked, name) {
+    console.log(checked, name)
```

Comment:
remove console

Reply:
This is in the storybook, it is logging just to demonstrate how the component works

Reply:
ok, fair enough